lib/secure_headers.rb

```diff
@@ -30,14 +39,60 @@ module SecureHeaders
   module Configuration
     class << self
```

Comment:
It seems a bit weird to me to be using class-level instance variables so much. That seems to be how it was before this PR, but at some point do you think it would be worthwhile to to refactor to have a Configuration instance that is passed around?